.github/workflows/release-push-maven-repo.yml

```diff
@@ -1,75 +0,0 @@
-# HMS Networks Solution Center
```

Comment:
can you add a description to the commit body on why this is deleted

Reply:
done